### PR TITLE
Migrate Mozilla Locamotion ExternalAccounts to new identifier.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
Reproduced/verified locally:
* Checkout to version ``2015-7.1``
* Add a ``LOCOMOTION`` ExternalAccount to your profile
* Checkout to master 
* Access your profile page
* Error gets raised
* Checkout to this PR
* Run migrations
* Access your profile page
* No error gets raised

@akatsoulas r?

File: mozillians/users/migrations/0074_fix_locomotion_type.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
```
Comment (akatsoulas) [inline mozillians/users/migrations/0074_fix_locomotion_type.py]: Add a couple of docstrings in forward and maybe backward methods.

**Comment by johngian** [comment]
@akatsoulas PR updated

**Comment by akatsoulas** [comment]
Tested locally and looks good. r+ :+1:

**Pull request merged**